docusaurus/docs/e2e-test-a-plugin/get-started.md

```diff
@@ -13,138 +13,28 @@ keywords:
 sidebar_position: 2
 ---
 
-import ScaffoldPluginE2ECreateDSPluginNPM from '@snippets/plugin-e2e-create-ds-plugin.npm.md';
-import ScaffoldPluginE2ECreateDSPluginYarn from '@snippets/plugin-e2e-create-ds-plugin.yarn.md';
-import ScaffoldPluginE2ECreateDSPluginPNPM from '@snippets/plugin-e2e-create-ds-plugin.pnpm.md';
-import ScaffoldPluginE2InstallNPM from '@snippets/plugin-e2e-install.npm.md';
-import ScaffoldPluginE2EInstallPNPM from '@snippets/plugin-e2e-install.pnpm.md';
-import ScaffoldPluginE2EInstallYarn from '@snippets/plugin-e2e-install.yarn.md';
 import ScaffoldPluginE2EStartGrafanaNPM from '@snippets/plugin-e2e-start-grafana.npm.md';
 import ScaffoldPluginE2EStartGrafanaPNPM from '@snippets/plugin-e2e-start-grafana.pnpm.md';
 import ScaffoldPluginE2EStartGrafanaYarn from '@snippets/plugin-e2e-start-grafana.yarn.md';
 import ScaffoldPluginE2ERunTestsNPM from '@snippets/plugin-e2e-run-tests.npm.md';
 import ScaffoldPluginE2ERunTestsPNPM from '@snippets/plugin-e2e-run-tests.pnpm.md';
 import ScaffoldPluginE2ERunTestsYarn from '@snippets/plugin-e2e-run-tests.yarn.md';
-import ScaffoldPluginE2EDSWorkflowNPM from '@snippets/plugin-e2e-ds-workflow.npm.md';
-import ScaffoldPluginE2EDSWorkflowYarn from '@snippets/plugin-e2e-ds-workflow.yarn.md';
-import ScaffoldPluginE2EDSWorkflowPNPM from '@snippets/plugin-e2e-ds-workflow.pnpm.md';
 
-This guide walks you through how to get started with end-to-end testing in your plugin with the `@grafana/plugin-e2e` tool. You will learn:
+This guide walks you through how to get started with end-to-end testing in your plugin with the `@grafana/plugin-e2e` tool.
 
-- How to install `@grafana/plugin-e2e`
-- How to configure authentication
-- How to write tests
-- How to setup a basic Github workflow that runs tests against multiple versions of Grafana
-
-:::warning
-The `@grafana/plugin-e2e` tool is still in beta and subject to breaking changes.
+:::note
+The `create-plugin` tool started scaffolding basic setup for `@grafana/plugin-e2e` from version 4.5.0. If you already have a plugin and it was scaffolded with an earlier version of the `create-plugin` tool, you may follow the [migration](./migrate-from-grafana-e2e.md) guide for instruction on how to install and configure `@grafana/plugin-e2e` manually.
 :::
 
 ## Before you begin
 
 You need to have the following:
 
-- Grafana [plugin development environment](https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment).
+- Grafana [plugin development environment](https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment)
```

Comment:
nit: full stop consistency [is preferred](https://grafana.com/docs/writers-toolkit/write/style-guide/style-conventions/#unordered-lists).
```suggestion
- Grafana [plugin development environment](https://grafana.com/developers/plugin-tools/get-started/set-up-development-environment).
```